webpack.config.js

```diff
@@ -253,6 +261,27 @@ module.exports = {
       ],
       ServiceWorker: {navigateFallbackURL: '/'},
     }),
+    new webpack.optimize.CommonsChunkPlugin({
+      name: 'vendor',
+      minChunks({ context }) {
+        if (context) {
+          const isNodeModule = context.indexOf('node_modules') !== -1;
+          const isBowerComponent = context.indexOf('bower_components') !== -1;
+          return isNodeModule || isBowerComponent;
+        }
+        return false;
```

Comment:
*Very* minor / I don’t feel strongly about this at all, but I would invert this and return `false` early if `context` is not truthy, so that the “meat” of the function body describes the expected case.